make/config.mk

```diff
@@ -109,6 +109,9 @@ USE_MKLDNN = 0
 # USE_NGRAPH_IE - If 1, then if/when nGraph is built it will contain support for inference-engine
 #   processing.
 #
+# USE_NGRAPH_GPU - If 1, then if/when nGraph is built it will contain support for NVIDIA GPU
+#   processing. Requires USE_CUDA and USE_CUDA_PATH to be set above.
+#
```

Comment:
@fengleitian's comment reminded me that we need a comment here explaining the flag, please see 
cb6dac5

Reply:
Probably a good idea.

Reply:
Is it possible to actually make that a hard requirement in the build system in ngraph-mxnet-bridge?

Reply:
Yeah, I didn't add any check there because it is a hard requirement in ngraph though.

Reply:
It will complain at build time if those criteria are not met, which is why I wasn't too worried about adding a comment about it.